Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/BaseShapeComponentMode.cpp

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "BaseShapeComponentMode.h"
+#include <AzCore/Component/NonUniformScaleBus.h>
+#include <AzToolsFramework/API/ComponentModeCollectionInterface.h>
+#include <AzToolsFramework/ActionManager/Action/ActionManagerInterface.h>
+#include <AzToolsFramework/ActionManager/HotKey/HotKeyManagerInterface.h>
+#include <AzToolsFramework/ActionManager/Menu/MenuManagerInterface.h>
+#include <AzToolsFramework/Manipulators/ShapeManipulatorRequestBus.h>
+
+namespace AzToolsFramework
+{
+    static constexpr AZStd::string_view EditorMainWindowActionContextIdentifier = "o3de.context.editor.mainwindow";
+    static constexpr AZStd::string_view EditMenuIdentifier = "o3de.menu.editor.edit";
```

Comment:
suggestion: I think there's a central location for these now, @AMZN-daimini might be able to point you in the right direction.